package.json

```diff
@@ -8,7 +8,8 @@
     "prettier:format": "prettier --write .",
     "prettier:check": "prettier --check .",
     "typecheck": "tsc --build",
-    "validate": "concurrently --kill-others-on-fail -g -p \"[{name}]\" -n \"lint,prettier,typecheck\" \"npm:lint\" \"npm:prettier:check\" \"npm:typecheck\""
+    "validate": "concurrently --kill-others-on-fail -g -p \"[{name}]\" -n \"lint,prettier,typecheck\" \"npm:lint\" \"npm:prettier:check\" \"npm:typecheck\"",
+    "test": "jest"
```

Comment:
Can we also add a `"test:watch": "jest --watch"` script to make it easier to run the tests in watch mode.

Reply:
done